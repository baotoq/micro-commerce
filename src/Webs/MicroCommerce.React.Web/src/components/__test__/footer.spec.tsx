import React from "react";
import renderer from "react-test-renderer";
import { render, screen } from "@testing-library/react";

import Footer from "../footer";

describe("Footer", () => {
  it("should match snapshot", () => {
    const component = renderer.create(<Footer />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it("should show app name", () => {
    render(<Footer />);
    expect(screen.getByText("Micro Commerce")).toBeInTheDocument();
  });
});
