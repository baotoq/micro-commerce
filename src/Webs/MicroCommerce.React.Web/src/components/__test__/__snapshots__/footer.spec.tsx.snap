// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should match snapshot 1`] = `
<footer
  className="makeStyles-footer-1"
>
  <div
    className="MuiContainer-root MuiContainer-maxWidthSm"
  >
    <p
      className="MuiTypography-root MuiTypography-body2 MuiTypography-colorTextSecondary"
    >
      Copyright Â© 
      <a
        className="MuiTypography-root MuiLink-root MuiLink-underlineHover MuiTypography-colorPrimary"
        href="/"
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onMouseEnter={[Function]}
      >
        Micro Commerce
      </a>
       
      2021
      .
    </p>
  </div>
</footer>
`;
